test/test.js

```diff
@@ -0,0 +1,9 @@
+const StellarSdk = require("@stellar/stellar-sdk");
+
+const secret = "SA22TDBINLZMGYUDVXGUP2JMYIQ3DTJE53PNQUVCDK73XRS6TDVYU7WW"; // Example secret
+try {
+  const keypair = StellarSdk.Keypair.fromSecret(secret);
+  console.log("Public Key:", keypair.publicKey());
+} catch (error) {
+  console.error("Error generating keypair:", error);
```

Comment:
There appears to not be a check for the failure occurring anywhere. If you can confirm with screenshots/output that there's no error occurring when running this test case, we can just consider the issue closed without a PR.